infrastructure/static/js/mm-webflow.js

```diff
@@ -686,7 +697,29 @@ function mmWebflow(js) {
       }
```

Comment:
Blank function?

Reply:
Looks fine to me, maybe Github minimized the code?

Reply:
Oh dumb me, thanks.